cmake/FindMyGUI.cmake

```diff
@@ -109,6 +109,12 @@ ELSE (WIN32) #Unix
     ENDIF(MYGUI_STATIC)
 ENDIF (WIN32)
 
+if (ANDROID AND MYGUI_STATIC)
+    if (NOT FREETYPE_LIBRARY_REL AND NOT FREETYPE_LIBRARY_DBG)
+        message(FATAL_ERROR "Freetype required for Android, but not found! Without freetype, fonts will not work !")
+    endif()
+endif()
```

Comment:
We do need this check whenever MYGUI_STATIC is true, not just for android.

Just add a REQUIRED to the find_package(Freetype) like so:
<pre>
find_package(Freetype REQUIRED)
</pre>
This way we get a cmake-generated error message if freetype is not found, making this whole block 
<pre>
+if (ANDROID AND MYGUI_STATIC)
 +    if (NOT FREETYPE_LIBRARY_REL AND NOT FREETYPE_LIBRARY_DBG)
 +        message(FATAL_ERROR "Freetype required for Android, but not found! Without freetype, fonts will not work !")
 +    endif()
 +endif()</pre>
no longer necessary.